Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -520,6 +520,41 @@ def fetch_incidents():
     return incidents
 
 
+def filter_results_by_fields(results, fields_to_view):
+    """
+    Filter a list of dictionaries by including only specified fields.
+
+    Parameters:
+        - results (list): A list of dictionaries representing rows of data.
+        - fields_to_view (list): A field names to include.
+
+    Returns:
+        list: A new list of dictionaries with only the specified fields.
+
+    Raises:
+        ValueError: If the fields_to_view contains invalid column names.
+    """
+    if not fields_to_view or not results:
```

Comment:
@manas-metron what happens if both of these parameters are empty ?

Reply:
"results" is not a parameter. "results" is the actual response from the devo.
If "fields_to_view" is not passed, then we just return the actual response.